travis/publish.sh

```diff
@@ -104,6 +104,50 @@ safe_checkout_master() {
   fi
 }
 
+test_server() {
+  # Test Stackdriver Storage module with the latest version of Zipkin server.
+  temp_dir=$(mktemp -d)
+  pushd $temp_dir
+
+  # Download and unpack Zipkin Server
+  wget https://jitpack.io/com/github/openzipkin/zipkin/zipkin-server/master-SNAPSHOT/zipkin-server-master-SNAPSHOT-exec.jar
+
+  # Zipkin server JAR is a self-executable with headers that causes unzip to exit with status 1
+  # Fix the archive first.
+  zip -A zipkin-server-master-SNAPSHOT-exec.jar
+  unzip zipkin-server-master-SNAPSHOT-exec.jar -d zipkin
+
+  # Unpack our module
+  unzip -n $TRAVIS_BUILD_DIR/autoconfigure/storage-stackdriver/target/*-module.jar -d zipkin
+
+  # Start the server. Note that the GOOGLE_APPLICATION_CREDENTIALS is configured from .travis.yml
+  STORAGE_TYPE=stackdriver
+  STACKDRIVER_PROJECT_ID=zipkin-gcp-ci
+  java -Dloader.path=stackdriver -Dspring.profiles.active=stackdriver -cp zipkin/ org.springframework.boot.loader.PropertiesLauncher &
+  ZIPKIN_PID=$!
+
+  echo "Waiting for Zipkin server to start..."
+  $TRAVIS_BUILD_DIR/travis/wait-for-it.sh localhost:9411 -t 60
+  exit_status=$?
+  if [ $exit_status -ne 0 ]; then
+    exit $exit_status
+  fi
+
+  health_check_result=$(curl --silent localhost:9411/actuator/health| jq -r .details.zipkin.status)
+
+  if [ "$health_check_result" != "UP" ]; then
+    echo "Health check failed!"
+    curl --silent localhost:9411/actuator/health | jq .
```

Comment:
Does the error message work as expected? The unquoted period in `jq .` could be interpreted by shell as a current directory. Maybe surround it with single quotes to guarantee no interpretation: ` jq '.'`.

Reply:
i think this works as expected. `jq -h` shows a similar usage:
```
$ echo '{"foo": 0}' | jq .
...
```

Once I fix the wait-for-it stuff, I'll do a test that should fail the health check and we can validate the error msg.

Reply:
your negative test worked. I removed it!